content/images/github.svg

```diff
@@ -1,4 +1,4 @@
-<!-- Modified from: https://fontawesome.com/icons/github -->
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512">
+<!-- Modified from: https://fontawesome.com/icons/github under its CC BY 4.0 License -->

```

Comment:
Nitpik: does it make more sense to remove the colon since it breaks up the sentence. Now might be a good time to make this change.

Reply:
I think you're right since we now have the license after the url as part of the same sentence. Changed it.